.github/workflows/tests.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will test this Node.js package.
+# For more information see: https://docs.github.com/en/actions/guides/building-and-testing-nodejs
+
+name: Test with NPM
+
+# Execute on every push, pull request and if activated manually.
+on: [push, pull_request, workflow_dispatch]
+
+# The caching settings were based on https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#example-using-the-cache-action
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
It looks like 15.x is EOLed for 2021-06-01. Really worth including in the tests?